src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/AttributesAccordion.tsx

```diff
@@ -14,26 +14,31 @@ import {
 import { getAttributesAccordionExpandedPanel } from './state/attributesAccordionSelector';
 import {
   setAttributesAccordionExpandedPanel,
-  fetchAttributes
+  fetchAttributes,
+  resetSelectedAttributes,
+  updateSelectedAttributes
 } from './state/attributesAccordionActions';
 
-import {
-  Genes,
-  Transcripts,
-  Variations,
-  Location,
-  Orthologues,
-  Phenotypes,
-  Paralogues
-} from './sections';
+import { Orthologues } from './sections';
+import customDownloadStorageService from 'src/content/app/custom-download/services/custom-download-storage-service';
 
+import ImageButton, {
+  ImageButtonStatus
+} from 'src/shared/image-button/ImageButton';
+import { ReactComponent as ResetIcon } from 'static/img/shared/trash.svg';
 import styles from './AttributesAccordion.scss';
 
+import AttributesAccordionSection from 'src/content/app/custom-download/containers/content/attributes-accordion/sections/AttributesAccordionSection';
+import JSONValue from 'src/shared/types/JSON';
+
 type Props = StateProps & DispatchProps;
 
 const AttributesAccordion = (props: Props) => {
   useEffect(() => {
     props.fetchAttributes();
+    props.updateSelectedAttributes(
+      customDownloadStorageService.getSelectedAttributes()
+    );
```

Comment:
Should reading from and writing to storage perhaps be moved to the redux layer?

Reply:
That's done.